#!/bin/sh /usr/share/dpatch/dpatch-run
## 08-init-lsb.dpatch by Michael Meskes <meskes@debian.org>
##
## DP: Make initscript lsb cmpliant.

@DPATCH@

diff -Naurp virtualbox-ose-1.6.0-dfsg.orig/src/VBox/Installer/linux/vboxnet.sh.in virtualbox-ose-1.6.0-dfsg/src/VBox/Installer/linux/vboxnet.sh.in
--- virtualbox-ose-1.6.0-dfsg.orig/src/VBox/Installer/linux/vboxnet.sh.in	2008-05-09 09:28:11.000000000 +0000
+++ virtualbox-ose-1.6.0-dfsg/src/VBox/Installer/linux/vboxnet.sh.in	2008-05-09 09:28:41.000000000 +0000
@@ -20,10 +20,10 @@
 #
 ### BEGIN INIT INFO
 # Provides:       vboxnet
-# Required-Start: $network
-# Required-Stop:
+# Required-Start: $remote_fs $network
+# Required-Stop:  $remote_fs
 # Default-Start:  2 3 4 5
 # Default-Stop:   0 1 6
 # Description:    VirtualBox permanent host networking setup
 ### END INIT INFO
 
@@ -50,14 +50,13 @@ if [ -z "$NOLSB" ]; then
     . /lib/lsb/init-functions
     fail_msg() {
         [ -f "$VARFILE" ] && rm $VARFILE
-        echo ""
-        log_failure_msg "$1"
+    	log_action_end_msg 1
     }
     succ_msg() {
-        log_success_msg " done."
+        log_action_end_msg 0
     }
     begin_msg() {
-        log_daemon_msg "$@"
+    	log_action_begin_msg $1
     }
 else
     if [ "$system" = "redhat" ]; then
